backend/src/main/scala/bloop/CompilerCache.scala

```diff
@@ -4,35 +4,22 @@ import java.io.File
 import java.lang.Iterable
 import java.io.PrintWriter
 import java.util.concurrent.ConcurrentHashMap
+
 import javax.tools.JavaFileManager.Location
 import javax.tools.JavaFileObject.Kind
-import javax.tools.{
-  FileObject,
-  ForwardingJavaFileManager,
-  StandardJavaFileManager,
-  ForwardingJavaFileObject,
-  JavaFileManager,
-  JavaFileObject,
-  JavaCompiler => JavaxCompiler
-}
-
+import javax.tools.{FileObject, ForwardingJavaFileManager, ForwardingJavaFileObject, JavaFileManager, JavaFileObject, StandardJavaFileManager, JavaCompiler => JavaxCompiler}
```

Comment:
Can you leave the imports as they were and disable the formatter in intellij when hacking on bloop? You should be using the scalafmt formatter instead.

Reply:
The formatter touches only the modified lines. But, yeah, I'll try to set up scalafmt.
Thanks for heads up.

Reply:
How do I run `sbt scalafmt` on a single file? It reformats many other files if invoked on the whole project. :(

Reply:
Try setting up scalafmt with IntelliJ. It has an option to reformat with save. Save only the file and it will be reformatted.